example/metro.config.js

```diff
@@ -1,21 +1,21 @@
-const { getDefaultConfig } = require('expo/metro-config');
-const path = require('path');
+import { getDefaultConfig } from 'expo/metro-config'
+import path from 'path'
```

Comment:
have you try to run the app after these changes? because metro will not support es6

Reply:
thanks- you are right, just updated